NebulaPatcher/Patches/Transpilers/BuildTool_Click_Transpiler.cs

```diff
@@ -0,0 +1,40 @@
+﻿using HarmonyLib;
+using NebulaWorld;
+using NebulaWorld.Factory;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+
+namespace NebulaPatcher.Patches.Transpiler
+{
+    [HarmonyPatch(typeof(BuildTool_Click))]
+    class BuildTool_Click_Transpiler
+    {
+        [HarmonyTranspiler]
+        [HarmonyPatch(nameof(BuildTool_Click.CreatePrebuilds))]
+        static IEnumerable<CodeInstruction> CreatePrebuilds_Transpiler(IEnumerable<CodeInstruction> instructions, ILGenerator iL)
```

Comment:
Do we need to check we are in an MP game here?

Reply:
FactoryManager.EventFromServer or FactoryManager.EventFromClient should only be true if we are in an MP game no?

Reply:
What will the value of `FactoryManager.PacketAuthor` be? Would it be a defined value?

Reply:
Or is always returning false what we want to do?

Reply:
Yes if it's false it continues normally, if it's true it skips code. So returning false in SP is fine